Matrix.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+__author__ = 'Yaron'
+
+
+class Matrix:
+    def __init__(self, tuples):
+        self.__tuples = tuples
+
+    def __len__(self):
+        return len(self.tuples)
+
+    def __repr__(self):
+        return 'Matrix('+str(self.tuples)+')'
```

Comment:
use f strings instead. 

We pretty much always use fstrings instead of concatenation.